src/scenes/Projects/Budget/ProjectBudget.tsx

```diff
@@ -57,7 +58,12 @@ export const ProjectBudget = () => {
   const { data, loading, error } = useProjectBudgetQuery({
     variables: { id: projectId },
   });
-  const [updateBudgetRecord] = useUpdateProjectBudgetRecordMutation();
+  const [
+    updateBudgetRecord,
+    { loading: isUpdateLoading },
```

Comment:
@frankyma Maybe we should play it safe and use the `onCompleted` method of the mutation to set `budgetRecordIdInUpdate.current` back to `null`; what do you think?

Reply:
Technically it shouldn't be necessary but if you insist on it I don't mind.  Lmk

Reply:
> Technically it shouldn't be necessary but if you insist on it I don't mind. Lmk

Eh, you're right. Just being overly cautious